client/src/charts/NivoCharts.js

```diff
@@ -42,58 +43,52 @@ const get_scale_bounds = (stacked, raw_data, zoomed) => {
+  </Fragment>
+);
+
 const default_tooltip = (tooltip_items, formatter, total) => ( // total indicates percent value tooltip being used
   <div style={{color: window.infobase_color_constants.textColor}}>
-    <table style={{width: '100%', borderCollapse: 'collapse', fontSize: window.innerWidth < breakpoints.minSmallDevice ? '14px' : 'inherit'}}>
+    <table className="nivo-tooltip">
       <tbody>
         { tooltip_items.map(
           tooltip_item => ( 
             <tr key = {tooltip_item.id}>
-              <td style= {{padding: '3px 5px'}}>
+              <td className="nivo-tooltip__content">
                 <div style={{height: '12px', width: '12px', backgroundColor: tooltip_item.color}} />
               </td>
-              { window.innerWidth < breakpoints.minSmallDevice ?
-                  <td>
-                    <div style={{padding: '3px 5px'}}> {tooltip_item.id} </div>
-                    <div style={{padding: '3px 5px'}} dangerouslySetInnerHTML={{__html: formatter(tooltip_item.value)}} />
-                  </td>
-                  :
-                  <Fragment>
-                    <td style={{padding: '3px 5px'}}> {tooltip_item.id} </td>
-                    <td style={{padding: '3px 5px'}} dangerouslySetInnerHTML={{__html: formatter(tooltip_item.value)}} />
-                  </Fragment>
-              }
-            </tr>
-          )
-        )}
-      </tbody>
-    </table>
-  </div>
-);
-
-const percent_value_tooltip = (tooltip_items, formatter, total) => (
-  <div>
-    <table style={{width: '100%', borderCollapse: 'collapse', fontSize: window.innerWidth < breakpoints.minMediumDevice ? '14px' : 'inherit'}}>
-      <tbody>
-        { tooltip_items.map(
-          tooltip_item =>(
-            <tr key = {tooltip_item.id}>
-              <td style = {{padding: '3px 5px'}}>
-                <div style = {{height: '12px', width: '12px', backgroundColor: tooltip_item.color}}/>
-              </td>
-              { window.innerWidth < breakpoints.minSmallDevice ?
-                  <td>
-                    <div style = {{padding: '3px 5px'}}>{tooltip_item.id}</div>
-                    <div style = {{padding: '3px 5px'}} dangerouslySetInnerHTML = {{__html: formatter(tooltip_item.value)}}/>
-                    <div style = {{padding: '3px 5px'}} dangerouslySetInnerHTML = {{__html: formats.percentage1(tooltip_item.value/total)}}/>
-                  </td>
-                  :
-                  <Fragment>
-                    <td style = {{padding: '3px 5px'}}>{tooltip_item.id}</td>
-                    <td style = {{padding: '3px 5px'}} dangerouslySetInnerHTML = {{__html: formatter(tooltip_item.value)}}/>
-                    <td style = {{padding: '3px 5px'}} dangerouslySetInnerHTML = {{__html: formats.percentage1(tooltip_item.value/total)}}/>
-                  </Fragment>
-              }
+              <MediaQuery minDeviceWidth={breakpoints.minSmallDevice}> {/* Have I gotten min and max mixed up here?? */}
```

Comment:
RE: comment, nope, got it right! In media queries, min means actual => min, so it's the right choice for selecting cases where the screen is larger than "small". As for the breakpoint, `minSmallDevice` is the correct option for a `minDeviceWidth` check.